assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -263,9 +263,11 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
       snapHorizontal={true}
       snapVertical={true}
       snapCenter={true}
-      horizontalGuidelines={[0, canvasHeight / 2, canvasHeight]}
-      verticalGuidelines={[0, canvasWidth / 2, canvasWidth]}
-      elementGuidelines={otherNodes}
+      horizontalGuidelines={
+        actionsEnabled && [0, canvasHeight / 2, canvasHeight]
```

Comment:
I think we also should add `|| null`. Not great to pass `false` to a property that expects an array.

Reply:
`Moveable` doesn't seem to crash with `null`. Passing an empty array now instead.

Reply:
Pretty sure `null` is fine. I was commenting about the `false` value before.

Reply:
I mis-typed; `Movable` **does** seem to crash when I tested with `null`. That's why I went with this.